aws-lambda-java-events/src/main/java/com/amazonaws/services/lambda/runtime/graalvm/LambdaEventsGraalVMFeature.java

```diff
@@ -0,0 +1,107 @@
+package com.amazonaws.services.lambda.runtime.graalvm;
+
+import org.graalvm.nativeimage.hosted.Feature;
+import org.graalvm.nativeimage.hosted.RuntimeReflection;
+
+import java.io.File;
+import java.io.IOException;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Enumeration;
+import java.util.List;
+
+/**
+ * The aws-lambda-java-events library supports GraalVM by containing a reflect-config.json file. This is located
+ * src/main/resources/META-INF/native-image/com.amazonaws/aws-lambda-java-events/reflect-config.json
+ *
+ * This config is used by the GraalVM native-image tool in order to load the required classes and methods into the
+ * native binary it creates.
+ *
+ * Any event or response class added to this library needs to be added to this config file.
+ *
+ * This implementation of Feature does that by iterating through the events package and registering
+ * them for runtime reflection.
+ */
+public class LambdaEventsGraalVMFeature implements Feature {
+
+    public static final String EVENTS_PACKAGE_NAME = "com.amazonaws.services.lambda.runtime.events";
```

Comment:
I think we should make this generic enough for it to be reused across all our libraries in this project, and move it out of the runtime addressable scope. It should be a build-only class that is not exposed in the library.

This is essentially boilerplate code that will traverse our libraries and determine what the appropriate `native-image` config is, so we don't really need a copy of it per project.